src/Bitmap_cubical_complex/include/gudhi/embedded_complex_release.hpp

```diff
@@ -0,0 +1,484 @@
+#include <gudhi/Bitmap_cubical_complex.h>
+
+#include <future>
+#include <thread>
+#include <chrono>
+
+#ifndef EMBEDDED_CPLX
+#define EMBEDDED_CPLX
+
+template <typename T>
+void print_vector(std::vector<T> vect){
+    if(vect.size() == 0){
+        std::cout << "[]\n";
+    }else{
+        std::cout << "[" << vect[0];
```

Comment:
Is there any reason why the `vect[0]` is printed outside of the loop?

Reply:
Since we want to display the vector with a comma between numbers, I think that either vect[0] or vect[vect.size()-1] must be printed outside the loop.